src/collection.js

```diff
@@ -57,13 +57,30 @@ export default class Collection {
 		this._idProperty = options && options.idProperty || 'id';
 
 		/**
-		 * A helper mapping external items from bound collection ({@link #bindTo})
-		 * and actual items of the collection.
+		 * A helper mapping external items of a bound collection ({@link #bindTo})
+		 * and actual items of this collection.
 		 *
 		 * @protected
 		 * @member {Map}
 		 */
-		this._boundItemsMap = new Map();
+		this._bindToExternalToInternalMap = new Map();
+
+		/**
+		 * A helper mapping items of this collection to external items of a bound collection
+		 * ({@link #bindTo}).
+		 *
+		 * @protected
+		 * @member {Map}
+		 */
+		this._bindToInternalToExternalMap = new Map();
```

Comment:
Also, it's not clear what these variable are used for. What is `external items`? What is `internal`? A longer description would be useful since there variable seems to be essential for the binding.